packages/aws-cdk/test/cli/telemetry/sanitation.test.ts

```diff
@@ -76,12 +76,35 @@ describe(sanitizeCommandLineArguments, () => {
     const argv = {
       _: ['deploy'],
       STACKS: ['MyStack'],
-      ['require-approval']: 'broadening',
       ['build-exclude']: ['something'],
     };
     expect(sanitizeCommandLineArguments(argv)).toEqual({
       path: ['deploy', '$STACK1'],
-      parameters: { 'require-approval': '<redacted>', 'build-exclude': '<redacted>' },
+      parameters: { 'build-exclude': '<redacted>' },
```

Comment:
How come?

Reply:
i removed `require-approval` from the test 'non-boolean options are redacted' and added it as a separate test called 'enum options are allowed'